docs/admin.rst

```diff
@@ -408,17 +408,28 @@ for how to enable error logging for the *Source Interface*.
 
 .. include:: includes/get-logs.txt
 
+.. _immediate_update:
+
 Immediately Apply a SecureDrop Update
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 SecureDrop will update and reboot once per day. However, if after a SecureDrop
 update `is announced`_ you wish to fetch the update immediately, you can SSH
-into each server and run:
+into each server (via ``ssh app`` and ``ssh mon``) and run:
 
 .. code:: sh
 
   sudo cron-apt -i -s
```

Comment:
As a side note, these switches are undocumented. Since we're deprecating cron-apt, not fleshing this out any further, but just in case of any suport questions about it:

`-i`: Run immediately
`-s`: Print to stdout